IdentityCore/src/oauth2/aad_base/MSIDAADWebviewFactory.m

```diff
@@ -91,25 +92,33 @@ - (MSIDWebviewSession *)embeddedWebviewSessionFromConfiguration:(MSIDWebviewConf
     
     MSIDWebviewSession *session = [[MSIDWebviewSession alloc] initWithWebviewController:embeddedWebviewController
                                                                                 factory:self
-                                                                            redirectUri:configuration.redirectUri
                                                                            requestState:state];
     return session;
 }
 
 #endif
 
 - (MSIDWebviewResponse *)responseWithURL:(NSURL *)url
+                            requestState:(NSString *)requestState
                                  context:(id<MSIDRequestContext>)context
                                    error:(NSError **)error
 {
     // Try to create a WPJ response
     MSIDWebMSAuthResponse *wpjResponse = [[MSIDWebMSAuthResponse alloc] initWithURL:url context:context error:nil];
     if (wpjResponse) return wpjResponse;
     
+    // Try to create a browser reponse
+    MSIDWebOpenBrowserResponse *browserResponse = [[MSIDWebOpenBrowserResponse alloc] initWithURL:url
+                                                                                          context:context
+                                                                                            error:nil];
```

Comment:
do we want to pass error?

Reply:
No, we fall back to the next case if there is an error.
We can log it though.